admin_guide/manage_users.adoc

```diff
@@ -0,0 +1,135 @@
+= Managing Users
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Adding Users
+
+Once new users log in to OpenShift, an account is created for that user and the
+cluster administrator can
+link:../admin_guide/manage_authorization_policy.html[manage the access level of
+each user].
+
+== Deleting Users
+
+=== Improper Deletion
+
+OpenShift user configuration is stored in several locations within OpenShift.
+Regardless of the identity provider, OpenShift internally stores details like
+role-based access control (RBAC) information and group membership. To completely
+remove user information, this data must be removed in addition to the user
+account.
+
+If an account is improperly deleted, the following type of error can be seen,
+even though the identification provider already shows the user *myuser* as
+existing:
+
+====
+----
+{
+    "MESSAGE": "E0412 06:07:48.142862     465 login.go:141] Unable to authenticate password: Error creating or updating mapping for: &api.DefaultUserIdentityInfo{ProviderName:\"htpasswd_auth\", ProviderUserName:\"myuser\", Extra:map[string]string{}} due to users \"myuser\" not found",
+    "PRIORITY": "6",
```

Comment:
We can probably trim this error message to just the "MESSAGE" here? I.e., just line 36.